pkg/process/executor.go

```diff
@@ -92,12 +98,12 @@ func (wx Server) loadModule(ctx context.Context, args api.Executor_spawn_Params)
 		return nil, err
 	}
 
-	hash := ByteCode(bc).Hash()
-	name := hex.EncodeToString(hash[:])
-
+	name := ByteCode(bc).String()
 	config := wazero.
 		NewModuleConfig().
-		WithName(name)
+		WithName(name).
+		WithEnv("WW_MODULE_CHECKSUM", name).
```

Comment:
I tend to move hard-coded values like this to a `consts.go` file, what are your thoughts on this?

Reply:
Done. Resolving.